wrapper-specification.md

```diff
@@ -338,6 +343,35 @@ Property Name|Type|Accessibility|Description
 ---|---|---|---
 Connected|Boolean|Read
 
+###Methods
+
+Method name | Return type | Arguments | Description
+---|---|---|---
+On    | void | None | Turns the led on by setting its brightness to the maximum level.
+Off   | void | None | Turns the led off.
+Flash | void | ON interval: Number, OFF interval: Number | Enables `timer` trigger and sets `delay_on` and `delay_off` attributes to the provided values (in milliseconds).
+
+###Static methods
```

Comment:
Not sure if the section name make sense for all of the supported languages. May be `Special functions` would be better.